pyblish_qml/qml/Perspective/Page.qml

```diff
@@ -126,6 +153,47 @@ Item {
         }
     }
 
+    CommentBox {
+        id: commentBox
+
+        visible: item.itemType == "instance"
+
+        //Enable editing only when the GUI is not busy with something else
+        readOnly: overview.state != ""
+        text: item.itemType == "instance" ? getComment() : ""
+
+        isUp: root.commenting
+
+        anchors {
+            bottom: footer.top
+            left: parent.left
+            right: parent.right
+            top: (isMaximised && height == parent.height - xfooter.height) ? tabBar.top : undefined
```

Comment:
Here actually have a typo, `xfooter`, which doesn't exists but yet the code works.
Looks like the value of `isMaximised` is always being `false`, which makes the following conditions as dead code. Fix coming.